package.json

```diff
@@ -1,107 +1,99 @@
+        "test:e2e": "vue-cli-service test:e2e",
+        "test:unit": "vue-cli-service test:unit"
+    },
+    "dependencies": {
+        "@apollo/client": "^3.2.2",
+        "@lumino/algorithm": "^1.3.3",
+        "@lumino/datagrid": "^0.14.0",
+        "@lumino/default-theme": "^0.5.0",
+        "@lumino/widgets": "^1.14.0",
+        "axios": "^0.20.0",
+        "axios-fetch": "^1.1.0",
+        "babel-runtime": "^6.26.0",
+        "core-js": "3",
+        "dedent": "^0.7.0",
+        "dotparser": "^1.0.0",
+        "enumify": "^2.0.0",
+        "graphiql": "^1.0.5",
+        "graphql": "^15.3.0",
+        "graphql-tag": "^2.11.0",
+        "jshint": "^2.12.0",
+        "lodash": "^4.17.20",
+        "nprogress": "^0.2.0",
+        "react": "^16.13.1",
+        "react-dom": "^16.13.1",
+        "vue": "^2.6.12",
+        "vue-i18n": "^8.21.1",
+        "vue-markdown": "^2.2.4",
+        "vue-meta": "^2.4.0",
+        "vue-router": "^3.4.5",
+        "vue-spinner": "^1.0.4",
+        "vue-the-mask": "^0.11.1",
+        "vuetify": "^2.3.12",
+        "vuex": "^3.5.1",
+        "vuex-router-sync": "^5.0.0"
+    },
+    "devDependencies": {
+        "@babel/plugin-proposal-class-properties": "^7.10.4",
+        "@cypress/code-coverage": "^3.8.1",
+        "@cypress/webpack-preprocessor": "^5.4.6",
+        "@mdi/js": "^5.6.55",
+        "@vue/babel-preset-app": "^4.5.6",
+        "@vue/cli-plugin-babel": "^4.5.6",
+        "@vue/cli-plugin-e2e-cypress": "^4.5.6",
+        "@vue/cli-plugin-eslint": "^4.5.6",
+        "@vue/cli-plugin-unit-mocha": "^4.5.6",
+        "@vue/cli-service": "^4.5.6",
+        "@vue/test-utils": "^1.1.0",
+        "babel-eslint": "^10.1.0",
+        "bufferutil": "^4.0.1",
+        "chai": "^4.2.0",
+        "cross-env": "^7.0.2",
+        "cross-fetch": "^3.0.6",
+        "eslint": "^7.10.0",
+        "eslint-config-standard": "^14.1.1",
+        "eslint-config-vuetify": "^0.5.0",
+        "eslint-plugin-import": "^2.22.1",
+        "eslint-plugin-node": "^11.1.0",
+        "eslint-plugin-standard": "^4.0.1",
+        "eslint-plugin-vue": "^7.0.0",
+        "istanbul-instrumenter-loader": "^3.0.1",
+        "istanbul-lib-coverage": "^3.0.0",
+        "null-loader": "^4.0.0",
+        "nyc": "^15.1.0",
+        "sass": "^1.26.11",
+        "sass-loader": "^10.0.2",
+        "sinon": "^9.1.0",
+        "standard": "^14.3.4",
+        "subscriptions-transport-ws": "^0.9.18",
+        "svgo": "^1.3.2",
+        "utf-8-validate": "^5.0.2",
+        "vue-cli-plugin-eslint-config-vuetify": "0.0.3",
+        "vue-cli-plugin-vuetify": "^2.0.7",
+        "vue-cli-plugin-vuetify-essentials": "^0.8.3",
+        "vue-template-compiler": "^2.6.12",
+        "vuetify-loader": "^1.6.0",
+        "webpack": "^4.44.2"
+    },
+    "resolutions": {
+        "istanbul-instrumenter-loader/**/istanbul-lib-instrument": "4.0.1"
+    },
+    "bugs": {
+        "url": "https://github.com/cylc/cylc-ui/issues"
+    }
```

Comment:
@datamel could you re-format with 2 spaces? No strong opinion on 2 vs 4, but it'd be easier to review with 2 to limit what's changed.

Reply:
Sorry about that @kinow, it wasn't passing my eslint without this but I've fixed it in vscode now :)